apps/web/src/components/ui/info.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react'
```

Comment:
Just curious in catching-up to Web a bit while walking through this change set, why ui and why *.jsx in this area?

Reply:
We are switching all components over to jsx, although I can't recall why Tibet has gone with this.

/ui is just the place where shadcn npx commands drops those components. Just helping us keep track of what's been pulled in from shadcn already. At some point we'll go through and weed out defunct components and decide if we want to hold on to the /ui

Reply:
because then the editor knows its jsx and not just js. its needed for certain linting rules e.g.

Reply:
Needless to day this is a long standing debate in the community of course, with the React team somewhat being officially not fans of JSX while not saying it loudly. I probably have a light preference for plain .js, but certainly not a strong one. I imagine in Vite it's helpful for quicker hot reloading. Good to know either way. Are you wanting us to do the same with mobile files? You can answer later, but the question is now in me :)

Reply:
interesting, I didnt know there was controversy. yes exactly for vote hot reloading each component needs to be in its own file with only 1 default export. and I have a listing rule for that but I want it to only loop at component files so thats why having them as .jsx makes things much easier.

Reply:
Yes, I remembered you'd saiid that and imagined this came from the same root... As for the controversy, it's really not that important, but since you kinda asked :). Enjoy the read 🫠 : https://github.com/airbnb/javascript/pull/985#issuecomment-239145468

Reply:
for fwiw, Dan now is working primarily on the React Native bluesky app, and all their files are .tsx